apps/frontend/src/app/3_organisms/GettingStartedPopup/GettingStartedPopup.module.css

```diff
@@ -55,7 +55,7 @@
 }
 
 .image {
-  @apply bg-gray-70 min-w-[4.375rem] h-[4.375rem] rounded mr-4 text-sovryn-blue;
+  @apply bg-gray-70 min-w-[4.375rem] h-[4.375rem] rounded mr-4 text-sovryn-blue flex items-center justify-center;
```

Comment:
This is not related to this selector but `.content` has set `margin-bottom: 4.5rem` which is too much compared to the design. I'd set it to let's say `2rem` (and I think it looks better with `2rem`).